srcpkgs/go/template

```diff
@@ -31,11 +32,37 @@ if [ "$CROSS_BUILD" ]; then
 	fi
 fi
 
+# Workaround for go-1.19 compilation error on i686
+#   runtime/cgo(.text): unknown symbol __stack_chk_fail_local in pcrel
+if [ "$XBPS_MACHINE" = i686 ]; then
+	export CGO_CFLAGS="-fno-stack-protector"
+fi
+
+post_extract() {
+	local patch patches
+	if [ "$XBPS_MACHINE" = i686 ]; then
+		patches+=" fix-i686-tests.patch"
+	fi
+	if [ "$XBPS_LIBC" = musl ]; then
+		patches+=" fix-musl-tests.patch"
+	fi
+	for patch in $patches; do
+		cp "$FILESDIR/$patch" "$wrksrc/$patch"
+	done
+	for patch in $patches; do
+		patch -Np1 <"$patch"
+	done
```

Comment:
this is ugly and the tests that fail on i686 probably also fail on other 32bit archs, I would just patch all of them out unconditionally.

Reply:
Done